src/Raven.Studio/typescript/models/resources/info/resourceInfo.ts

```diff
@@ -139,8 +159,11 @@ abstract class resourceInfo {
 
             return currentResource.qualifiedName === this.qualifiedName;
         });
-    }
 
+        this.totalSizeToShow = ko.pureComputed(() => {
+            return this.totalSize() === "-1 Bytes" ? "0 Bytes" : this.totalSize();
```

Comment:
Can we have both totalSize and totalsize as properties of this object and compare this totalSize but not formatted as string, to avoid magic constants?